Frontend/src/register/pages/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styles from './index.module.scss'
+import RegistrationList from '../components/registration_list'
+import RegistrationPanel from '../components/registration_panel'
+
+export default function App() {
+  return (
+    <div className={styles.container}>
+      <RegistrationPanel> </RegistrationPanel>
```

Comment:
Nit: you could just use a self closing tag `<RegistrationPanel />`

Reply:
One of the react plugins should be able to enforce this.